src/main/java/org/spongepowered/api/command/conversation/Conversation.java

```diff
@@ -0,0 +1,268 @@
+import org.spongepowered.api.event.cause.conversation.ConversationEndType;
+import org.spongepowered.api.plugin.PluginContainer;
+import org.spongepowered.api.text.Text;
+
+import java.util.Optional;
+import java.util.Set;
+
+import javax.annotation.Nullable;
+
+/**
+ * Represents a created conversation with one or more {@link Conversant}s.
+ *
+ * <p>These are checked every five minutes to see if there are no players.
+ * If this is true, the conversation is ended with </p>
+ */
+public interface Conversation {
+
+    /**
+     * A convenience method to get the id of the conversation archetype.
+     *
+     * @return The id of the conversation archetype
+     */
+    default String getId() {
+        return getArchetype().getId();
+    }
+
+    /**
+     * Gets who started the conversation, if present.
+     *
+     * @return The {@link PluginContainer} who started this conversation
+     */
+    PluginContainer getCreator();
+
+    /**
+     * Gets a data container where you can store all relevant information you
+     * gather from each question's {@link AnswerHandler}.
+     *
+     * @return The overall context of this conversation
+     */
+    DataContainer getContext();
+
+    /**
+     * Gets the handler which handles input to see if it should end
+     * the {@link Conversation}.
+     *
+     * @return The cancelling handler
+     */
+    default CancellingHandler getCancellingHandler() {
+        return getArchetype().getCancellingHandler();
+    }
+
+    /**
+     * An ease of use method to retrieve the banner for each question, which
+     * is set if the title is specified and has padding added on.
+     *
+     * @return The conversation header, if available
+     */
+    default Optional<Text> getBanner() {
+        return getArchetype().getBanner();
+    }
+
+    /**
+     * An ease of use method to retrieve the header for each question, which
+     * is not always set.
+     *
+     * <p>Occurs after the banner if that is set.</p>
+     *
+     * @return The conversation header, if available
+     */
+    default Optional<Text> getHeader() {
+        return getArchetype().getHeader();
+    }
+
+    /**
+     * Gets the archetype of this conversation. Contains initial information.
+     *
+     * @return The initial archetype of this conversation
+     */
+    ConversationArchetype getArchetype();
+
+    /**
+     * Gets every {@link Conversant} within this conversation.
+     *
+     * @return A set of all the {@link Conversant}s
+     */
+    Set<Conversant> getConversants();
+
+    /**
+     * Gets the question that is currently being accessed or processed.
+     *
+     * @return The current question
+     */
+    Optional<Question> getCurrentQuestion();
+
+    /**
+     * Sets the new question, sending them the header, as well as the prompt
+     * of the question.
+     *
+     * @param question The question to set to
+     */
+    void setQuestion(@Nullable Question question);
```

Comment:
Thinking about it now, plugins shouldn't really need to set a question in the first place, that should all be handled by the implementation with the answer handlers. How would you feel about me removing this method from the API completely?